shared/profile/index.js

```diff
@@ -84,68 +82,71 @@ class Profile extends Component<void, ?Props, void> {
   }
 }
 
-const connector: TypedConnector<TypedState, TypedDispatch<Action>, OwnProps, ?Props> = new TypedConnector()
```

Comment:
I ticketed this. Our typedconnector passes the entire state into the help function which stops connect from early outting when our subsection of the store isn't changing. we can maybe have a different typedconnector but for now just using plain connect